tools/object-storage/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/prometheus/busybox:latest
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
+
+COPY object-storage /bin/object-storage
```

Comment:
Do you think this name describes what this tool does?

To me, it is a block syncer. It's not an object storage.